### ISSUE TITLE
problem with installation

### ISSUE THREAD

**Issue by ssemigr** [joshp23/ttrss-to-wallabag-v2#15]
I installed the plugin following the instructions, but plugin is not listed in the list of user plugins. 
There are no errors. The version of ttrss is  v17.1 (60e97d9).
I tried to put the plugin both in plugins and plugins.local folders, and the permissions are fine (browser is able to open the files in the plugin folder).
Any hint how to debug it?

**Comment by joshp23** [comment]
that's a head scratcher if permissions are set right... 

Try using developer mode in Chrome or FF and see if there is a hint.

**Comment by ssemigr** [comment]
I found out what was the problem: the fact that the name of the plugin folder was hardcoded in a couple of places. To make everything consistent, I renamed the plugin folder and plugin name to wallabag, changed the name of the class to Wallabag, and made the corresponding changes  in a couple of places in init.php and in one place in wallabag_v2.js, and now everything works fine.

I'm closing the issue, but perhaps an info in README about the name of the plugin folder would be helpful to others?
